### PR TITLE
Handle broken dependencies

### PR DESCRIPTION
**Pull request by mgax**
Fixes https://github.com/hoover/snoop2/issues/39

### CODE REVIEW

File: snoop/data/digests.py [hunk 1]
```diff
@@ -38,15 +41,25 @@ def gather(blob, collection_pk, **depends_on):
 
     tika_rmeta_blob = depends_on.get('tika_rmeta')
     if tika_rmeta_blob:
-        with tika_rmeta_blob.open(encoding='utf8') as f:
-            tika_rmeta = json.load(f)
-        rv['text'] = tika_rmeta[0].get('X-TIKA:content', "")
+        if isinstance(tika_rmeta_blob, ShaormaBroken):
+            rv['broken'].append(tika_rmeta_blob.reason)
+            log.warn("tika_rmeta task is broken; skipping")
```
Comment (gabriel-v) [inline snoop/data/digests.py]: This means a lot of log traffic. Don't think it's of any use, you should see the `broken` list in the digest, right?

Comment (mgax) [reply snoop/data/digests.py]: Right. How about I change those to `log.debug`?

Comment (gabriel-v) [reply snoop/data/digests.py]: sure

Comment (gabriel-v) [reply snoop/data/digests.py]: Merge it afterwards

**Pull request merged**